R/tostring.R

```diff
@@ -60,22 +275,25 @@ do_cell_fnotes_wrap <- function(mat, widths, max_width, tf_wrap) {
     unlist(mapply(wrap_string,
       str = mfs,
       width = cell_widths_mat,
-      collapse = "\n"
+      collapse = "\n",
+      MoreArgs = list(fontspec = fontspec)
     )),
     ncol = ncol(mfs)
   )
 
-  ## XXXXX this is wrong and will break for listings cause we don't know when
-  ## we need has_topleft to be FALSE!!!!!!!!!!
-  mat <- mform_handle_newlines(mat)
+  if(expand_newlines) {
+    ## XXXXX this is wrong and will break for listings cause we don't know when
+    ## we need has_topleft to be FALSE!!!!!!!!!!
+    mat <- mform_handle_newlines(mat)
```

Comment:
I think it is set as FALSE right?

Reply:
Yeah this issue has been solved for a long time, I think. We have accessors that can tell us whether we have topleft material (in practice)